YOUR_ADMIN/includes/classes/observers/auto.whos_online_ip_blocker_link.php

```diff
@@ -20,6 +20,12 @@ protected function update(&$class, $eventID, $item, &$additional_ipaddress_links
     {
         if (empty($item)) return;
 
-        $additional_ipaddress_links .= ' &mdash; <a href="' . zen_href_link(FILENAME_WHOS_ONLINE, zen_get_all_get_params(array('ip', 'action')) . 'action=block&ip=' . $item['ip_address']) . '">' . IP_BLOCKER_TEXT_BLOCK_IP . '</a>';
+        if (function_exists('ip_blocker_block) && ip_blocker_block($item['ip_address'])) {
+            $link = '[IP IS BLOCKED]';
```

Comment:
Could you add this language text to the plugin's /admin/includes/languages/extra_definitions file?

Reply:
Done